docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-class-issuers.md).
+
+To update credit class metadata, run the following command:
+
+```sh
+regen tx ecocredit update-class-metadata [class-id] [new-metadata]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-class-metadata.md).
+
+## Project
+
+Now that we have a credit class, we can create a project within the credit class. Only an account listed as a credit class issuer can create a project within the credit class.
+
+A project represents a group or individual providing ecosystem services. The ecological outcomes of a project are measured and verified and then credits are issued by a credit class issuer.
+
+For more information about projects, see [Ecocredit Concepts](../../modules/ecocredit/01_concepts.md).
+
+### Create Project
+
+We are now ready to create a project and to use the IRI we generated earlier in this tutorial. The IRI for the project can be used in the next command as the value of `[metadata]`.
+
+To create a project, run the following command:
+
+```sh
+regen tx ecocredit create-project [class-id] [jurisdiction] [metadata]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_create-project.md).
+
+### Update Project
+
+Only the admin of a project can update the project. The account that created the project (i.e. the credit class issuer) is assigned as the admin upon creation. After the project has been created, the admin can update the project, including the account assigned as the admin.
+
+To update a project admin, run the following command:
+
+```sh
+regen tx ecocredit update-project-admin [project-id] [new-admin]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-project-admin.md).
+
+To update project metadata, run the following command:
+
+```sh
+regen tx ecocredit update-project-metadata [project-id] [new-metadata]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-project-metadata.md).
+
+## Credit Batch
+
+Now that we have a project, we can create a credit batch and issue credits. Only an account listed as a credit class issuer can create a credit batch for projects within the credit class.
+
+A credit batch is a vintage of credits for a given project and at the time of creation, credits are issued to a list of recipients, which can include accounts representing buffer pools.
+
+For more information about credit batches, see [Ecocredit Concepts](../../modules/ecocredit/01_concepts.md).
+
+### Create Batch
+
+We are now ready to create a credit batch and to use the IRI we generated earlier in this tutorial. The IRI for the credit batch can be used in the next command as the value of `[metadata]`.
```

Comment:
For batches, there isn't a `[metadata]` argument in the CLI command, but instead it's a field in the batch-json.  Could you change the wording here to communicate that?

batch-json:
```json
{
  "project_id": "C01-001",
  "issuer": "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw",
  "issuance": [
    {
      "recipient": "regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw",
      "tradable_amount": "1000"
    },
    {
      "recipient": "regen1depk54cuajgkzea6zpgkq36tnjwdzv4ak663u6",
      "retired_amount": "1000",
      "retirement_jurisdiction": "US-OR",
      "retirement_reason": "offsetting electricity consumption"
    }
  ],
  "metadata": "regen:13toVgf5UjYBz6J29x28pLQyjKz5FpcW3f4bT5uRKGxGREWGKjEdXYG.rdf",
  "start_date": "2020-01-01T00:00:00Z",
  "end_date": "2021-01-01T00:00:00Z"
}
```

Reply:
Updated https://github.com/regen-network/regen-ledger/pull/1965/commits/c5863d7f0fb55eab13f5cc7bfa0a86da841d5aa7